helix-core/src/main/java/org/apache/helix/controller/stages/AttributeName.java

```diff
@@ -40,5 +40,8 @@
   PipelineType,
   LastRebalanceFinishTimeStamp,
   ControllerDataProvider,
-  STATEFUL_REBALANCER
+  STATEFUL_REBALANCER,
+
+  /** Event should be processed by the expected session. */
```

Comment:
This claim is not related to this field but with the internal logic. Let's just say this is the session that in which we get the event.